.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+* @ncksllvn @jbalboni
```

Comment:
@jbalboni In the event I'm not around and there is something coming through, I added you as a reviewer. I hope this is okay.

Reply:
Can we also add James? Is there anyone else on the website team who might be helpful to add?

Reply:
Hm, rethinking this, I don't think we want to be notified for every content PR. This might be a lot in our inbox. I'm going to reduce the scope for now to just the `pages` directory and the content folks.